README.md

```diff
@@ -1,3 +1,17 @@
+## Quick reference
+
+Maintained by: [the ROOT team](https://root.cern/) and collaborators
+
+Where to get help: [the ROOT forum](https://root-forum.cern.ch/)
+
+## Supported tags and respective Dockerfile links
+
+* [`latest`](https://github.com/root-project/root-docker/blob/6.20.06-fedora32/fedora/Dockerfile), [`6.20.06-fedora32`](https://github.com/root-project/root-docker/blob/6.20.06-fedora32/fedora/Dockerfile)
```

Comment:
I think that having latest point to something not using 6.22 can be confusing. Maybe it's time to switch to a different image for this tag?

Reply:
it's temporary. if we make it point to something else, next release maybe _that_ will take more time to move to 6.22.

Reply:
please open an issue if you really think we should switch, merging this for now as this PR is orthogonal to what we use as `latest`